flink-utils/src/main/java/org/hypertrace/core/flinkutils/avro/RegistryBasedAvroSerde.java

```diff
@@ -63,11 +63,7 @@ public boolean isEndOfStream(T nextElement) {
 
   @Override
   public byte[] serialize(T element) {
```

Comment:
some javadoc about the caller getting exceptions from the `KafkaAvroSerializer.serialize()`? Otherwise, caller wouldn't even know what they get.

Reply:
added